Objects/TI4_PurgeBag.ttslua

```diff
@@ -0,0 +1,16 @@
+--- Announces purged objects.
+-- @author Darrell (copied from Deleted Items)
+-- @author Jabberwocky
+--
+
+function onObjectEnterContainer(bag, enterObject)
+    if bag == self then
+        printToAll('PURGING ' .. enterObject.getName())
```

Comment:
Good idea.  Maybe I'll set these to an interesting color.

Reply:
That would be nice. Should probably also debounce the reporting to reduce spammy messages.